data/styles/clear/include/Basemap.mapcss

```diff
@@ -433,9 +433,16 @@ area|z10-[landuse=farmyard],
 
 /* Parking */
 
-area|z15-[amenity=parking]
+area|z15-[amenity=parking],
 {fill-color: @parking;}
 
+area|z15-[amenity=parking][parking=underground],
+area|z15-16[amenity=parking][access=private],
+area|z15-16[amenity=parking][parking=lane],
+area|z15-16[amenity=parking][parking=street_side],
+{fill-opacity: 0;}
```

Comment:
let's wait for the https://github.com/organicmaps/organicmaps/pull/5789 to merge and use a new way to reset

Reply:
now using the new way, will regenerate rebase and generate styles once it's merged 👍